TECH_DEBT.md

```diff
@@ -0,0 +1,10 @@
+# Technical debt to pay back
+
+* We added a new custom Logger to service-front, to enable us to log as JSON and log MVC error events in more detail. Consequently, we should remove opg-lpa-logger from service-front/composer.json, as it's no longer used. However, removing this and updating composer.lock causes the app to break. Need to investigate why that is and upgrade composer packages.
```

Comment:
Will we raise tickets for these tech debt items?

Reply:
I may just do that and remove this document actually. I already raised the issue for this.

Reply:
I added a commit to remove this document and added issues for the tech debt.